dpnp/dpnp_iface_mathematical.py

```diff
@@ -1814,13 +1815,39 @@ def sum(
 
     """
 
+    if axis is not None:
+        if not isinstance(axis, (tuple, list)):
+            axis = (axis,)
+
+        axis = normalize_axis_tuple(axis, x.ndim, "axis")
+
     if out is not None:
         pass
     elif initial != 0:
         pass
     elif where is not True:
         pass
     else:
+        if axis == (0,) and len(x.shape) == 2:
```

Comment:
Looks you missed to support `dtype` and `keepdims` input arguments or to exclude invoking the extension when they are passed,

Reply:
Done